elbepack/init/Makefile.mako

```diff
@@ -134,6 +134,23 @@ run-con:
 		-nographic \
 		-smp $(SMP)
 
+
+run_qemu:
+	$(INTERPRETER) -M $(MACHINE) \
+		$(INTERPRETER-ARGS) \
+		-nographic \
+		-monitor unix:qemu-monitor-socket,server,nowait \
+		-serial unix:vm-serial-socket,server,nowait \
+		-device virtio-rng-pci \
+		-device virtio-net-pci,netdev=user.0 \
+		-drive file=$(INITVM),if=$(HD_TYPE),bus=1,unit=0 \
+		-no-reboot \
+		-netdev user,ipv4=on,id=user.0,hostfwd=tcp::7587-:7588${fwd} \
```

Comment:
This should default to 7588 as the libvirt based one.
Otherwise the default config won't work.

Reply:
To my understanding it's the same for the libvirt VM, and it's working for me with the default config. Please see:
https://github.com/Linutronix/elbe/blob/0494866f13a20374514e056391138199803f0060/elbepack/init/libvirt.xml.mako#L35
https://github.com/Linutronix/elbe/blob/0494866f13a20374514e056391138199803f0060/elbepack/config.py#L12

Reply:
Indeed. No clue what I did wrong yesterday. Sorry for the noise.

Reply:
No problem. You can take a look at https://github.com/tomirgang/elbe_dev_container/tree/main/.devcontainer if you want to see the feature in action.

Reply:
Thanks!